README.md

```diff
@@ -1,23 +1,63 @@
-<div align="center">
+<h1 align="center">
   <a href="https://github.com/denysdovhan/spaceship-prompt">
     <img alt="spaceship →~ prompt" src="https://cloud.githubusercontent.com/assets/3459374/21679181/46e24706-d34b-11e6-82ee-5efb3d2ba70f.png" width="400">
   </a>
-</div>
+  <br>🚀⭐ Spaceship ZSH <br>
+</h1>
+
+<h4 align="center">
+  <a href="http://zsh.org" target="_blank"><code>Zsh</code></a> prompt for Astronauts.
+</h4>
+
+<p align="center">
+  <!-- NPM Version -->
+    <a href="https://npmjs.org/package/spaceship-prompt">
+    <img src="https://img.shields.io/npm/v/spaceship-prompt.svg?style=flat-square"
+      alt="NPM Version" />
+  </a>
+
+  <a href="https://travis-ci.org/denysdovhan/spaceship-prompt">
+    <img src="https://img.shields.io/travis/denysdovhan/spaceship-prompt.svg?style=flat-square"
+      alt="CI Status" />
+  </a>
 
-# 🚀⭐ Spaceship ZSH
+  <a href="http://zsh.org/">
+    <img src="https://img.shields.io/badge/zsh-%3E%3Dv5.2-777777.svg?style=flat-square"
+      alt="Zsh Version" />
+  </a>
 
-[![NPM version][npm-image]][npm-url]
-[![Build status][ci-image]][ci-url]
-[![Zsh][zsh-image]][zsh-url]
-[![Donate with card][donate-card-image]][donate-readme]
-[![Donate with Bitcoin][donate-btc-image]][donate-readme]
-[![Donate with Ethereum][donate-eth-image]][donate-readme]
+  <a href="#donate">
+    <img src="https://img.shields.io/badge/$-donate-ff69b4.svg?style=flat-square"
+      alt="Donate" />
+  </a>
+</p>
 
-> A [Zsh][zsh-url] prompt for Astronauts.
+<div align="center">
+  <h4>
+    <a href="https://denysdovhan.com/spaceship-prompt/">Website |</a>
```

Comment:
I would suggest removing the `|` dividers from the `<a>` tags. The underline behavior is unexpected with the dividers within the tag.

![image](https://user-images.githubusercontent.com/4658208/44735887-af36a380-aabb-11e8-8d81-edd8c95486ca.png)